src/data/code-samples/en/accessibility-announcement/jetpack-compose.md

```diff
@@ -0,0 +1,16 @@
+# Accessibility announcement - Android
+
+In Jetpack Compose, to notify `Composable` state changes, you can use the [`liveRegion`](https://developer.android.com/reference/kotlin/androidx/compose/ui/semantics/package-summary.html#(androidx.compose.ui.semantics.SemanticsPropertyReceiver).liveRegion()) property from [`semantics`](https://developer.android.com/reference/kotlin/androidx/compose/ui/semantics/package-summary#(androidx.compose.ui.Modifier).semantics(kotlin.Boolean,kotlin.Function1)) block modifier. You can choose from two options for `liveRegion`: [`LiveRegionMode.Polite`](https://developer.android.com/reference/kotlin/androidx/compose/ui/semantics/LiveRegionMode#Polite()) , which waits for the speech announcement in progress to complete, and [`LiveRegionMode.Assertive`](https://developer.android.com/reference/kotlin/androidx/compose/ui/semantics/LiveRegionMode#Assertive()), which interrupts ongoing speech to immediately announce changes. If you don't specify the `liveRegion` property, it indicates to Compose that updates to this field would not be announced.
```

Comment:
Place enter before 'You can choose', and use numbered list for the two options

Reply:
Added numbered list, and improved formatting